engine/src/main/java/org/hibernate/search/v6poc/engine/SearchMappingRepository.java

```diff
@@ -10,9 +10,6 @@
 import org.hibernate.search.v6poc.engine.impl.SearchMappingRepositoryBuilderImpl;
 import org.hibernate.search.v6poc.entity.mapping.spi.MappingKey;
 
-/**
- * @author Yoann Rodiere
- */
```

Comment:
I don't understand why you remove these tags now that they are everywhere. It's really not helping when reviewing.

Reply:
I'm only removing them when I made useful changes to the same file.

Reply:
I guess this one slipped in because I reverted some changes... Sorry about that.

Reply:
If you think they should go away, let's remove them once and for all in a single commit. But we should do it after the Lucene merge because I added the tags there too.

Personally, I don't have an issue with them: we maintain the `@author` tags for HV and I don't think it's that useless.

Reply:
> If you think they should go away, let's remove them once and for all in a single commit. But we should do it after the Lucene merge because I added the tags there too.

Ok, I'll do that.